<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWin</class>
 <widget class="QWidget" name="MainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alarms and Clock</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QTabWidget" name="tabWidge">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_timer">
      <attribute name="title">
       <string>Timer</string>
      </attribute>
      <widget class="QLabel" name="lblshow1_2">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>180</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell Condensed</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Select start to run an Timer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="image_clock_2">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>30</y>
         <width>161</width>
         <height>131</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:30px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>assets/img/timer.png</normaloff>assets/img/timer.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>120</width>
         <height>120</height>
        </size>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinbox_ht">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>220</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinbox_mt">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>220</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinbox_st">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>220</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_stopwatch_timer">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>261</width>
         <height>141</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="text">
        <string>00:00:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_stopwatch_stoptimer">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>240</y>
         <width>81</width>
         <height>81</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"> border-radius: 30px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>assets/img/stop-circle-256.png</normaloff>assets/img/stop-circle-256.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>42</width>
         <height>42</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_stopwatch_starttimer">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>160</y>
         <width>81</width>
         <height>81</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"> border-radius: 30px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>assets/img/Start.png</normaloff>assets/img/Start.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_stopwatch_pausetimer">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>240</y>
         <width>81</width>
         <height>81</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"> border-radius: 30px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>assets/img/Startt.png</normaloff>assets/img/Startt.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>45</width>
         <height>45</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>220</y>
         <width>16</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Algerian</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>220</y>
         <width>16</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Algerian</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>:</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Alarm</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="2" column="0">
          <widget class="QPushButton" name="image_clock">
           <property name="styleSheet">
            <string notr="true">border-radius:30px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>assets/img/Clock0.webp</normaloff>assets/img/Clock0.webp</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="leftMargin">
            <number>90</number>
           </property>
           <property name="topMargin">
            <number>60</number>
           </property>
           <property name="rightMargin">
            <number>90</number>
           </property>
           <property name="bottomMargin">
            <number>60</number>
           </property>
           <item>
            <widget class="QSpinBox" name="spinbox_h">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>31</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinbox_m">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>31</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_addalarm">
             <property name="maximumSize">
              <size>
               <width>45</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"> border-radius: 30px;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>assets/img/plus-circle-256.png</normaloff>assets/img/plus-circle-256.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>45</width>
               <height>45</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Stopwatch</string>
      </attribute>
      <widget class="QPushButton" name="btn_stopwatch_start">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>170</y>
         <width>81</width>
         <height>81</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"> border-radius: 30px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>assets/img/Start.png</normaloff>assets/img/Start.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_stopwatch_pause">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>250</y>
         <width>81</width>
         <height>81</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"> border-radius: 30px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>assets/img/Startt.png</normaloff>assets/img/Startt.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>45</width>
         <height>45</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_stopwatch_stop">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>250</y>
         <width>81</width>
         <height>81</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"> border-radius: 30px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>assets/img/stop-circle-256.png</normaloff>assets/img/stop-circle-256.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>45</width>
         <height>80</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_stopwatch">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>261</width>
         <height>141</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="text">
        <string>00:00:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_record">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>260</y>
         <width>71</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> border-radius: 30px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>assets/img/flag.png</normaloff>assets/img/flag.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>80</height>
        </size>
       </property>
      </widget>
      <widget class="QScrollArea" name="scrollArea">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>90</y>
         <width>111</width>
         <height>241</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poor Richard</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(143, 143, 143);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>111</width>
          <height>241</height>
         </rect>
        </property>
        <widget class="QWidget" name="gridLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>91</width>
           <height>211</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gl_record"/>
        </widget>
       </widget>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>40</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed</family>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(122, 122, 122);</string>
       </property>
       <property name="text">
        <string>Laps</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>63</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed</family>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(122, 122, 122);</string>
       </property>
       <property name="text">
        <string>Spilit</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
